src/Stripe.net.Tests/charges/test_data/stripe_charge_capture_options.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace Stripe.Tests.test_data
```

Comment:
Would you be willing to either (a) move these tests to the XUnit project or (b) write some more tests for the XUnit project to include here?  We're going to be removing the mspec tests.  :)